src/codegeneration/ModuleTransformer.js

```diff
@@ -276,8 +276,8 @@ export class ModuleTransformer extends TempVarTransformer {
     assert(this.moduleName);
     let name = tree.token.processedValue;
     // import/module {x} from './name.js' is relative to the current file.
-    let normalizedName = System.normalize(name, this.moduleName);
-    return parseExpression `System.get(${normalizedName})`;
+    let normalizedName = $traceurRuntime.normalizeModuleName(name, this.moduleName);
```

Comment:
Ok, I guess the answer to my question is "No, but SystemJS will use the correct runtime for each module format."

Up until now we provided our users with a loader automatically; in other words we force our loader on them.

Now we are headed towards letting users choose; in other words we force them to think about and configure the runtime.  I'm trying to find a way to make this painless and backcompat for existing users.